lighthouse-core/gather/gatherers/dobetterweb/document-event-listeners.js

```diff
@@ -77,7 +90,7 @@ class PageLevelEventListeners extends Gatherer {
           // Debugger.scriptParsed event so we get .url and other
           // needed properties.
           const combo = Object.assign(listener, script);
-          combo.objectId = location; // One of LISTENER_LOCATIONS.
+          combo.objectId = results.tagName;
```

Comment:
this is your object, right? doesnt seem right to call it objectId if its a tag name.